wallet/src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -857,7 +857,8 @@ deriveSafeCopySimple 0 'base ''WebTypes.CHash
 deriveSafeCopySimple 0 'base ''WebTypes.CId
 deriveSafeCopySimple 0 'base ''WebTypes.Wal
 deriveSafeCopySimple 0 'base ''WebTypes.Addr
-deriveSafeCopySimple 0 'base ''BackupPhrase
```

Comment:
hm, why do we store backup phrase at all :/ I doesn't make sense to keep this peace of info on the disk. (will check later why is this so)

EDIT: this was a mistake and is not used in acid (and it doesn't make sense to be used)